### DISCUSSION TITLE
我曾想做的平行线 - 没有秃头基因的燊的博客

### DISCUSSION THREAD

**Discussion by giscus** [yzs020220/giscus-repo#6] [Announcements]
# 我曾想做的平行线 - 没有秃头基因的燊的博客

也许在读计算机这个专业前，每个人都有想过自己以后做的事，我曾想做一个网站或APP，叫平行线。在这里，大家不会相互评论，没有那么多社交压力。

https://yzs020220.github.io/posts/12027/

**Comment by STEVE-1936550490** [comment]
:smiley:

**Comment by HCY-ASLEEP** [comment]
👋

**Comment by NCfish** [comment]
😎
